pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -88,6 +88,21 @@ def on_run_start(  # type: ignore[override]
         # add the previous `fetched` value to properly track `is_last_batch` with no prefetching
         data_fetcher.fetched += self.batch_progress.current.ready
 
+        dataloader_idx = f"_{kwargs['dataloader_idx']}" if "dataloader_idx" in kwargs else ""
+        if self.trainer.testing:
+            suffix = f"test_dataloader{dataloader_idx}_next"
```

Comment:
In result logging, we use the convention `dataloader_idx_{}`. We should do the same here too